src/FileSystemUtilities.js

```diff
@@ -21,7 +21,7 @@ export default class FileSystemUtilities {
 
   @logger.logifyAsync()
   static mkdirp(filePath, callback) {
-    mkdirp(filePath, callback);
+    mkdirp(filePath, { fs }, callback);
```

Comment:
Passing `graceful-fs` to `mkdirp` is an effort to minimize race conditions when conducting tons of parallel filesystem operations.

(These were especially noticeable because jest by default tries to run as many tests in parallel processes as possible)